packages/scandipwa/src/component/ProductList/ProductList.component.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Please remove this line, since:
1. We shouldn't disable eslint for such issues.
2. I don't see any unused variables in this PR.